app/views/sessions/new.html.haml

```diff
@@ -6,3 +6,4 @@
         = f.text_field :email, placeholder: "Email"
         = f.password_field :password, placeholder: "Password"
         = f.submit "Login"
+        = link_to('Forgot Password', new_password_reset_path)
```

Comment:
Mike, this will need some love. I played with it but it just a link below the login button